exercises/strain/canonical-data.json

```
@@ -0,0 +1,155 @@
+{
+  "exercise": "strain",
+  "cases": [
+    {
+      "description": "filter list returning only values that satisfy the predicate function",
+      "cases": [
+        {
+          "uuid": "26af8c32-ba6a-4eb3-aa0a-ebd8f136e003",
+          "description": "keep on empty list returns empty list",
+          "property": "keep",
+          "input": {
+            "list": [],
+            "predicate": "fn(x) -> true"
+          },
+          "expected": []
+        },
+        {
+          "uuid": "f535cb4d-e99b-472a-bd52-9fa0ffccf454",
+          "description": "keeps everything",
+          "property": "keep",
+          "input": {
+            "list": [1, 3, 5],
+            "predicate": "fn(x) -> x % 2 == 1"
```

Maybe do:

```suggestion
            "predicate": "fn(x) -> true"
```

I think this makes it more straightforward to see that is keeps everything.